views/webs/WebsView.js

```diff
@@ -719,6 +719,16 @@
         return null;
     }
 
+    function acceptableDarkLineClr ( initClr ) {
+        if (initClr == "LightSkyBlue" || initClr == "HotPink") {
+            return initClr;    
+        }
+        let theClr = initClr;
+        if (ColourArray.indexOf(initClr) > -1) {
```

Comment:
No need to to do 2 `.indexOf` operations when just one would suffice. :)
`        const clrIdx = ColourArray.indexOf(initClr);
        if (clrIdx > -1) {
            theClr = LineColourArray[clrIdx];
        }
`